examples/arm/executor_runner/CMakeLists.txt

```diff
@@ -59,8 +59,74 @@ endif()
 
 if(TARGET_BOARD STREQUAL "corstone-300")
   add_subdirectory(${ETHOS_SDK_PATH}/core_platform/targets/corstone-300 target)
+  target_compile_definitions(ethosu_target_common INTERFACE
+      # Configure NPU architecture timing adapters
+      # Ethos_U55_High_End_Embedded
+      # This is just example numbers and you should make this match your hardware
```

Comment:
Nit: add a reference url to figure out what is what? And what these number means? i.e. RLATENCY_0=32, which cycles we are talking about etc.